clients/src/main/java/org/apache/kafka/common/protocol/ApiKeys.java

```diff
@@ -415,4 +444,10 @@ public void visit(Type field) {
         return hasBuffer.get();
     }
 
+    public static Set<ApiKeys> enabledApis() {
```

Comment:
nit: if this is only used for unit tests, better add a one-liner comment for that.